input/docs/cs/classification_of_numbers_with_mnist.md

```diff
@@ -0,0 +1,42 @@
+
+Title: Klasifikace čísla pomocí natrénované sítě 
+-
+---
+
+ Description: MNIST je velká databáze ručně psaných číslic, které se použivají pro školení různých systémů ve zpracování obrazu a taktéž ke školení a testování v oblasti strojového učení.
+ 
+---
+>Tento dokument pracuje s obrázkem:
+[Obrázek číslice](.../media/trojka.png) v proměnné `img3`
+
```

Comment:
chtělo by to nějaký obrázek výsledku. Není to moc jednoznačné, protože výsledek se vypisuje do konzole. Zkus třeba fci insertText..